js/src/sdk/client/types.gen.ts

```diff
@@ -1063,16 +1699,39 @@ export type DirectExecuteReqDto = {
 };
 
 export type ActionExecutionResDto = {
+  /**
+   * The response data returned by the action execution.
+   */
   data: {
     [key: string]: unknown;
   };
+  /**
+   * The error message, if the action failed to execute. If the action is successful, this will be null.
+   */
   error?: string;
-  successfull?: string;
+  /**
+   * Whether the action execution was successfully executed or not. If this is false, error field will be populated with the error message.
+   * @deprecated
+   */
+  successfull: boolean;
```

Comment:
The `successfull` field (with double 'l') is marked as deprecated but still required in the type definition. This could cause confusion. Consider:
1. Making it optional if it's deprecated
2. Adding a migration guide for users to switch to `successful`
3. Adding a timeline for when `successfull` will be removed